packages/js-dpp/lib/identity/IdentityFactory.js

```diff
@@ -164,4 +169,13 @@ class IdentityFactory {
   }
 }
 
+/**
+ * @typedef {Object} PublicKeyOptions
```

Comment:
Not sure, it's "options" since they are not optional. "configs"?

Reply:
Renamed